zebedee-cms/src/main/java/com/github/onsdigital/zebedee/filters/AuthenticationFilter.java

```diff
@@ -66,35 +100,69 @@ public boolean filter(HttpServletRequest request, HttpServletResponse response)
             return true;
         }
 
-        // Pass through without authentication for login requests:
-        // Password requests check
         Path path = Path.newInstance(request);
+    
         if (noAuthorisationRequired(path)) {
             return true;
         }
 
         // Check all other requests:
         boolean result = false;
-        try {
-            Session session = Root.zebedee.getSessions().get(request);
-
-            if (session == null) {
-                forbidden(response);
-            } else {
-                result = true;
+        if (jwtSessionsEnabled) {
+            try {
+                //ensure that authorisation header present
+                if (request.getHeader("Authorization") == null) {
+                    unauthorisedRequest(response, NO_AUTH_HEADER_FOUND);
+                } else {
+                    try {
+                        sessions.set(request.getHeader("Authorization"));   
+                        result = true;
+                    } catch (SessionsStoreException e) {
+                        // treat access token expired or malformed access token as unauthorised
+                        if (e.getMessage() == ACCESS_TOKEN_EXPIRED_ERROR || e.getMessage() == TOKEN_NOT_VALID_ERROR ) {
+                            unauthorisedRequest(response, e.getMessage());
+                        } else {
+                            accessTokenError(response, e.getMessage());
+                        }
+                    }
+                }
+            } catch (IOException e) {
+                error(response);
+            }
```

Comment:
Can this be its own method, would help with the flow and readability in my opinion.